kubernetes/samples/charts/apache-samples/custom-sample/README.md

```diff
@@ -78,7 +78,7 @@ PathTrim /weblogic2
 </Location>
 ```
 
-* Place the `custom_mod_wl_apache.conf` file in a local directory `<host-config-dir>` on the host machine.
+* Create  PV / PVC (pv-claim-name) that can be used to place the `custom_mod_wl_apache.conf`.  Refer to the [Sample for creating a PV or PVC](/kubernetes/samples/scripts/create-weblogic-domain-pv-pvc/README.md).
```

Comment:
Create  PV / PVC -> Create  a PV / PVC
that can be used to place the `custom_mod_wl_apache.conf` -> not sure what you mean by this phrase, "can be used to place" -> do you mean, "can be used to locate" or "can be used to store"? Not sure which is correct, but "can be used to place" is not.

Reply:
Updated.